public/javascript/main.js

```diff
@@ -376,5 +376,21 @@ $(function(){
 
           $(this).data('chart', chart);
       }
-  })
+  });
+
+  $("a.download-with-term").on("click", function(e){
+    var term = $(e.target).data("term");
+    var house = $(e.target).data("house");
+    if (term && house) {
+        // inject query vars into URL before the target anchor
+        e.preventDefault();
+        location = $(e.target).attr('href')
+          .replace('#', "?" + $.param({term: term, house: house}) + "#");
+    }
+  });
+
+  if (typeof page_init_function === 'function') {
+      page_init_function();
```

Comment:
The javascript naming convention everywhere else in this project is camel case. Is there a reason to use underscores here instead?